numba/cuda/cudadrv/nvvm.py

```diff
@@ -295,12 +296,15 @@ def get_supported_ccs():
     if cudart_version_major == 0:
         _supported_cc = ()
     elif cudart_version_major < 9:
-        # CUDA 8.x
-        _supported_cc = (2, 0), (2, 1), (3, 0), (3, 5), (5, 0), (5, 2), (5, 3), (6, 0), (6, 1), (6, 2) # noqa: E501
-    elif cudart_version_major < 10:
+        _supported_cc = ()
+        ctk_ver = f"{cudart_version_major}.{cudart_version_minor}"
+        msg = f"CUDA Toolkit {ctk_ver} is unsupported by Numba - 9.0 is the " \
+              + "minimum required version."
+        warnings.warn(msg)
```

Comment:
Is this on the import path for `numba.cuda`? If so perhaps don't use `fstrings` as the error message gets lost for users on python's with no fstring support (e.g. someone accidentally using the unsupported Python 2 or <=3.5).

Reply:
It's not on the import path - if it we're we'd have the much bigger problem of #6149. (fixed in #6147)